src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -124,12 +167,58 @@ export default class ReplyField extends React.Component<Props, State> {
         setCursorPosition(editorState.getSelection().getFocusOffset());
     };
 
+    stopDefaultEvent = (e: React.KeyboardEvent): void => {
+        e.preventDefault();
+        e.stopPropagation();
+    };
+
+    setPopupListActiveItem = (index: number): void => this.setState({ activeItemIndex: index });
```

Comment:
We're using a lot of local state. What happens if the component is re-rendered due to a zoom or resize event?

Reply:
> We're using a lot of local state. What happens if the component is re-rendered due to a zoom or resize event?

The active selection index will reset to 0, which is not too bad. Again, we have to store the active index somewhere and this file is the only place, because it's the common parent of both Editor and PopList.